enterprise/server/remote_execution/snaploader/snaploader.go

```diff
@@ -639,6 +639,7 @@ func (l *FileCacheLoader) CacheSnapshot(ctx context.Context, key *fcpb.SnapshotK
 		ar.OutputFiles = append(ar.OutputFiles, out)
 		eg.Go(func() error {
 			ctx := egCtx
+			fileName := filepath.Base(filePath)
```

Comment:
how bout calling this fileType and adding a comment that it should be one of those 5 names you listed?